tests/mocks/AccessControl.cairo

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: MIT
+
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.starknet.common.syscalls import get_caller_address
+from openzeppelin.access.accesscontrol import AccessControl
+
+@constructor
+func constructor{
+        syscall_ptr : felt*,
+        pedersen_ptr : HashBuiltin*,
+        range_check_ptr
+    }(admin: felt):
+    AccessControl.initializer()
+    AccessControl._grant_role(0, admin)
```

Comment:
is this the desired default behavior? should it be part of the initializer?

Reply:
Its common, but nor default